src/deepsparse/v2/operators/operator.py

```diff
@@ -90,21 +93,39 @@ def __call__(
         else:
             inference_input = kwargs
 
-        run_output = self.run(inp=inference_input, context=context)
-
-        if self.has_output_schema():
-            return self.output_schema(**run_output)
-        return run_output
+        run_output, state_update = self.run(
+            inp=inference_input,
+            context=context,
```

Comment:
any reason we're still keeping context?

Reply:
I think the context right now contains the history of the past executed operators.

Reply:
removed